### DISCUSSION TITLE
Review of draft Project Proposal

### DISCUSSION THREAD

**Discussion by himanshu1409** [camicroscope/GSOC#32] [Q&A]
Hey @birm, my draft project proposal for project idea [4] Collection and Study Management is ready but it is filled with comments where I would like to get some guidance and insights from my mentor. So, how can I contact my potential mentor @nanli-emory ?

**Comment by nanli-emory** [comment]
Hi @himanshu1409, thanks for your message, and sorry for the delay in responding. We are receiving a high volume of very general questions, and we are responding to project-specific questions only.
